<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EthernetPortWidget</class>
 <widget class="QWidget" name="EthernetPortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>网口参数</string>
   </property>
   <widget class="QLabel" name="IP_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器端口号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>99</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>99</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>99</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>99</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>109</y>
      <width>10</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>109</y>
      <width>10</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>139</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>网口模式:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="EthernetPortcbB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>21</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMacthionAddr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>59</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备地址:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxTest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>网口测试</string>
   </property>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecv">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRecv">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSend">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="StartpushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
